src/index.ts

```diff
@@ -112,7 +112,27 @@ export interface BlockTime {
   timestamp: number;
 }
 
+export interface TransactionInput {
+  previous_output: string;
+  script_sig: string;
+  sequence: number[][];
+  witness: number[][];
+}
+
+export interface TransactionOutput {
+  value: number;
+  script_pubkey: string;
+}
+
+export interface Transaction {
+  version: number;
+  lock_time: number;
```

Comment:
not a huge deal but do you think we should map everything to camelCase like we do elsewhere in this lib?

Reply:
Yeah definitely, I totally forgot about that. Fixed in 5e632fd.